core/src/main/scala/org/apache/spark/storage/StorageUtils.scala

```diff
@@ -47,24 +50,30 @@ class StorageStatus(val blockManagerId: BlockManagerId, val maxMem: Long) {
   private val _nonRddBlocks = new mutable.HashMap[BlockId, BlockStatus]
 
   /**
-   * Storage information of the blocks that entails memory, disk, and off-heap memory usage.
+   * Storage information of the blocks that entails on-heap memory, off-heap memory and disk usage.
    *
    * As with the block maps, we store the storage information separately for RDD blocks and
    * non-RDD blocks for the same reason. In particular, RDD storage information is stored
-   * in a map indexed by the RDD ID to the following 4-tuple:
+   * in a map indexed by the RDD ID to the following 3-tuple case class:
    *
    *   (memory size, disk size, storage level)
```

Comment:
this is out of date now that you have a separate case class.  You could just get rid of this comment describing it since the names on the case class are pretty clear.